src/otk/document.py

```diff
@@ -49,10 +49,15 @@ def ensure(cls, deserialized_data: dict[str, Any]) -> None:
             raise ParseVersionError(f"omnifest must contain a key by the name of {NAME_VERSION!r}")
 
         # no toplevel keys without a target or an otk directive
-        targetless_keys = [key for key in deserialized_data
-                           if not key.startswith(PREFIX)]
+        targetless_keys = [
+            f" * \"{key}\" in {deserialized_data[key].get_annotation('src')}"
```

Comment:
If I drop in a key, `goobers:` into the top level I get a traceback from this:

```
  File "/root/otk/src/otk/document.py", line 38, in __init__
    Omnifest.ensure(tree)
  File "/root/otk/src/otk/document.py", line 53, in ensure
    f" * \"{key}\" in {deserialized_data[key].get_annotation('src')}"
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get_annotation'
```
so it probably also needs a None check :)

Reply:
I ran all possible tests and examples - how did you do that?